src/Applications/LDAS_App/ldas_setup

```diff
@@ -1531,26 +1504,27 @@ def _printRmInputKeys(rqdRmInpKeys, optRmInpKeys):
     print '# REQUIRED inputs'
     print '#'
     print '# NOTE:'
-    print '# - rm_name    = resource manager name'
-    print '#                  [At NCCS: Use "SLURM"; PBS is no longer used.'
-    print '# - account    = computational project number'
+    print '# - account      = computational project number'
     print '#                  [At NCCS: Use command "getsponsor" to see available account number(s).]'  
-    print '# - walltime   = walltime requested; format is HH:MM:SS (hours/minutes/seconds)'
-    print '# - ntasks     = number of processors requested'
+    print '# - walltime     = walltime requested; format is HH:MM:SS (hours/minutes/seconds)'
+    print '# - ntasks_model = number of processors requested for the model (output server is not included)'
+    print '# - ntasks-per-node  = number of tasks per node(  >28 , sky will be allocated, <=28 sky or hasw'
     print '#'
     for key in rqdRmInpKeys:
-        print key.ljust(23), ':'
+        print key + ':'
     print
     print '#'
     print '# OPTIONAL inputs'
     print '#'
     print '# NOTE:'
     print '# - job_name   = name of experiment; default is "exp_id"'
-    print '# - constraint = type of processors requested; default is no constraint.'
     print '# - qos        = quality-of-service; do not specify by default; specify "debug" for faster but limited service.'
+    print '# - oserver_nodes    = number of nodes for oserver ( default is 1 ), set it to 0 expicilty if no output server needed'
+    print '# - writers-per-node = tasks per oserver_node for writing ( default is 5 ),'
+    print '#                      total_writers = writers-per-node * oserver_nodes >=2 '
```

Comment:
Is "total_writers >= 2" a requirement?
What happens if a user specifies oserver_nodes = 1 and writers-per-node = 1 ??

Reply:
Yes, that is required. In your example, the program will hang. In the output server, it is implemented as captain-soldier paradim.

Reply:
I updated the help message.